Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions. So, chi square test is applicable.
<br>
Solution:- 
H0 (There is no association between male-female buyer rations across regions).
<br>
Ha (There is association between male-female buyer rations across regions).

In [47]:
from scipy import stats as st
import pandas as pd
import numpy as np

In [48]:
Buyer = pd.read_csv("BuyerRatio.csv",index_col = 0)

In [49]:
Buyer.head()

East  West  North  South
Observed Values                          
Males              50   142    131     70
Females           435  1523   1356    750

In [58]:
stat, p, dof, exp = st.chi2_contingency(Buyer)
print(stat,"\n",p,"\n",dof,"\n",exp) #o/p --> (Chi2 stats value, p_value, df, expected obsvations)

1.595945538661058 
 0.6603094907091882 
 3 
 [[  42.76531299  146.81287862  131.11756787   72.30424052]
 [ 442.23468701 1518.18712138 1355.88243213  747.69575948]]


In [70]:
print('''stat:- {0}\nP_value:- {1}\ndf:- {2}\nexp:- {3}'''.format(stat,p,dof,exp))

stat:- 1.595945538661058
P_value:- 0.6603094907091882
df:- 3
exp:- [[  42.76531299  146.81287862  131.11756787   72.30424052]
 [ 442.23468701 1518.18712138 1355.88243213  747.69575948]]


Here,  Statistics = 1.595945538661058
<br>
Chi square table(df = 3, Alpha = 0.05):- critical value = 7.82
<br>
Statistics < 7.82 --> fail to Reject H0
<br>
Hence, There is no association between male-female buyer rations across regions.